src/explore-education-statistics-admin/src/pages/release/ReleaseSummaryPage.tsx

```diff
@@ -39,35 +39,24 @@ const ReleaseSummaryPage = () => {
             These details will be shown to users to help identify this release.
           </p>
 
+          <h3>Publication details</h3>
           <SummaryList>
             <SummaryListItem term="Publication title">
               {publication.title}
             </SummaryListItem>
+            <SummaryListItem term="Lead statistician">
```

Comment:
These changes probably break a bunch of UI tests. Would run all of them and fix any failures!